### PR TITLE
Iam/kms key ring solution

### PR DESCRIPTION
**Pull request by trodge**

### CODE REVIEW

File: config-connector/solutions/iam/kpt/kms-key-ring/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  kms-key-ring
+# SYNOPSIS
+  Config Connector compatible yaml files for applying IAM roles to a kms key ring
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/kms-key-ring kms-key-ring
+  ```
+# REQUIREMENTS
+  A working cluster with Config Connector installed managing a project with Cloud Key Management Service API enabled.
```
Comment (maqiuyujoyce) [inline config-connector/solutions/iam/kpt/kms-key-ring/README.md:14]: A working cluster with Config Connector installed `and configured to manage` a project with 
`Cloud Key Management Service (KMS) API` enabled.

Let's be clear that it is Config Connector that has been configured and used to manage resources in projects. And also use the matched API name on Cloud Console.

File: config-connector/solutions/iam/kpt/kms-key-ring/iampolicymember.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicyMember
+metadata:
+  name: ring-iam-member
+spec:
+  member: ${IAM_MEMBER?}
```
Comment (maqiuyujoyce) [inline config-connector/solutions/iam/kpt/kms-key-ring/iampolicymember.yaml:19]: Missing setter for the member. Could you test before commit?

File: config-connector/solutions/iam/kpt/kms-key-ring/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+KMS Key Ring
+==================================================
+
+# NAME
+  kms-key-ring
+# SYNOPSIS
+  Config Connector compatible yaml files for applying IAM roles to a kms key ring
```
[the remark this answers is not in the document]
Comment (trodge) [reply config-connector/solutions/iam/kpt/kms-key-ring/README.md]: Change made

File: config-connector/solutions/iam/kpt/kms-key-ring/Kptfile [hunk 4]
```diff
@@ -0,0 +1,29 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: kms-key-ring
+packageMetadata:
+  shortDescription: apply roles to a kms key ring
```
[the remark this answers is not in the document]
Comment (trodge) [reply config-connector/solutions/iam/kpt/kms-key-ring/Kptfile]: Done

File: config-connector/solutions/iam/kpt/kms-key-ring/iampolicymember.yaml (hunk 2 above)
Comment (trodge) [reply config-connector/solutions/iam/kpt/kms-key-ring/iampolicymember.yaml]: added

File: config-connector/solutions/iam/kpt/kms-key-ring/README.md (hunk 1 above)
Comment (trodge) [reply config-connector/solutions/iam/kpt/kms-key-ring/README.md]: change made

**Pull request merged**